### PR TITLE
docs: optimized Chinese document for installation

### PR DESCRIPTION
**Pull request by jiawei666**
Optimized Chinese document for installation

### CODE REVIEW

**Comment by jiawei666** [comment]
The installation steps of Chinese document are lack of `vue cli` installation guide，so I added them

File: docs/zh/installation.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ Vue.use(VueRouter)
 
 如果使用全局的 script 标签，则无须如此 (手动安装)。
 
+## Vue CLI
+
+如果你有一个正在使用[Vue CLI](https://cli.vuejs.org/zh/)的项目，你可以添加Vue Router作为项目插件。你可以让CLI为你生成上面的代码以及两个示例路由。**它也会覆盖你的 `App.vue`** 。因此，请确保在项目中运行以下命令之前备份这个文件：
```
Comment (Jinjiang) [inline docs/zh/installation.md]:
```suggestion
如果你有一个正在使用 [Vue CLI](https://cli.vuejs.org/zh/) 的项目，你可以以项目插件的形式添加 Vue Router。CLI 可以生成上述代码及两个示例路由。**它也会覆盖你的 `App.vue`** ，因此请确保在项目中运行以下命令之前备份这个文件：
```

1. 对格式进行了调整
2. 在无歧义的基础上简化了描述

Comment (jiawei666) [reply docs/zh/installation.md]: 还有个地方疏漏了
```markdown
## Vue CLI 建议改成  ### Vue CLI （标题大小保持一致）
```

Comment (jiawei666) [reply docs/zh/installation.md]: 改动已提交

File: docs/zh/installation.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ Vue.use(VueRouter)
 
 如果使用全局的 script 标签，则无须如此 (手动安装)。
 
+### Vue CLI
+
+如果你有一个正在使用 [Vue CLI](https://cli.vuejs.org/zh/) 的项目，你可以以项目插件的形式添加 Vue Router。CLI 可以生成上述代码及两个示例路由。**它也会覆盖你的 `App.vue`** ，因此请确保在项目中运行以下命令之前备份这个文件：
```
Comment (Jinjiang) [inline docs/zh/installation.md]:
```suggestion
如果你有一个正在使用 [Vue CLI](https://cli.vuejs.org/zh/) 的项目，你可以以项目插件的形式添加 Vue Router。CLI 可以生成上述代码及两个示例路由。**它也会覆盖你的 `App.vue`**，因此请确保在项目中运行以下命令之前备份这个文件：
```

**Review by Jinjiang — APPROVED** [review]
@posva approved with a tiny format issue only. thanks.

**Pull request merged**